src/libs/RoomNameInputUtils.js

```diff
@@ -3,15 +3,13 @@ import CONST from '../CONST';
 /**
  * Modifies the room name in the following ways:
  * - Replaces spaces with dashes
```

Comment:
Ahh, just one small change -
```suggestion
 * Replaces spaces with dashes
```

Reply:
@thesahindia I think we can just use 
/**
 * Replaces spaces with dashes
 * @param {String} roomName
 * @returns {String}
 */
 
 as now there is only one way exist

Reply:
let me do it @thesahindia

Reply:
Done @thesahindia